README.md

```diff
@@ -18,32 +18,83 @@ The formatting and reannotation are based on information extracted from:
 - the 'filtered-results.csv' helper file as supplied to @sneumann during the [HUPO-PSI](http://www.psidev.info/hupo-psi-meeting-2020)
 
 
-Viewing the ISA-tab formatted PXD017710 with [ISATab-Viewer](https://sneumann.github.io/PXD017710/isaviewer-demo.html)
+Viewing the ISA-tab formatted and reannotated PXD017710 with [ISATab-Viewer](https://isa-tools.org/PXD017710/isaviewer-demo.html)
```

Comment:
I think we should stick to one location, happy to hand over the repo to isa-tools.org/PXD017710/